cmd/kubemark/hollow-node.go

```diff
@@ -88,6 +89,8 @@ func (c *hollowNodeConfig) addFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&c.ProxierMinSyncPeriod, "proxier-min-sync-period", 0, "Minimum period that proxy rules are refreshed in hollow-proxy.")
 	bindableNodeLabels := cliflag.ConfigurationMap(c.NodeLabels)
 	fs.Var(&bindableNodeLabels, "node-labels", "Additional node labels")
+	bindableRegisterWithTaints := cliflag.ConfigurationMap(c.RegisterWithTaints)
+	fs.Var(&bindableRegisterWithTaints, "register-with-taints", "Adding Taints")
```

Comment:
Please change the help message to something like: "Register the hollow node with the given list of taints"